fecgov/fec-cms
# Add document date range and penalty by respondent range filters

## Summary (required)

- Resolves https://github.com/fecgov/fec-cms/issues/6585

Add document date range for AO, AFs, ADRs, and MURs and penalty amount range filters for AFs, ADRs, and MURs

### Required reviewers

1 UX
1 Front end

## Impacted areas of the application

General components of the application that this PR will affect:

- AO
- Admin Fines
- ADR
- MUR

## Screenshots
### AOs
<img width="299" alt="AOs" src="https://github.com/user-attachments/assets/eb21fc26-2b1c-48da-a672-11699c887c57">

### MURs
<img width="198" alt="murs" src="https://github.com/user-attachments/assets/570fafe7-4518-4b79-b925-4f78c38ffd88">

### ADRs
<img width="268" alt="adrs" src="https://github.com/user-attachments/assets/d9824cc6-37f7-4ea4-84cf-975dc75c7d79">

### Admin Fines
<img width="268" alt="admin-fines" src="https://github.com/user-attachments/assets/8c6478e7-df2f-4582-8efb-43a94992a845">

## Related PRs

Related PRs against other branches:

branch | PR
------ | ------
feature/6006-move-adr-citations-to-dispositions | [link](https://github.com/fecgov/openFEC/pull/6023)
feature/add-document-date-filter | [link](https://github.com/fecgov/openFEC/pull/6015)

## How to test

- Checkout and run this branch
- Go to these search pages and make sure the penalty by respondent and document date range filters work
   - AO: http://localhost:8000/data/legal/search/advisory-opinions/
   - Admin Fines: http://localhost:8000/data/legal/search/admin-fines/
   - ADR: http://localhost:8000/data/legal/search/adrs/
   - MUR: http://localhost:8000/data/legal/search/murs/
